### DISCUSSION TITLE
Monterey compatible?

### DISCUSSION THREAD

**Discussion by ItsDan001** [MultimediaLucario/Lenovo-ThinkPad-T440s#3] [Q&A] [answered]

**Comment by MultimediaLucario** [comment]
I'm still trying to figure out Monterey compatibility myself.

**Comment by MultimediaLucario** [comment]
It is now compatible with macOS Monterey. Please keep in mind that Bluetooth does not work out of the box yet. I am currently trying to rectify that issue.

**Comment by MultimediaLucario** [comment, marked as answer]
macOS Monterey is fully funcitional! 😎

**Reply by kurtper** [reply]
Hi! i have a problem with battery icon. I have Thinkpad L440 Monterey 12.6.1
